packages/forma-36-react-components/src/components/ControlledInput/ControlledInput.tsx

```diff
@@ -69,35 +75,51 @@ export const ControlledInput = (props: ControlledInputPropTypes) => {
   );
 
   return (
-    <input
-      className={classNames}
-      value={value}
-      name={name}
-      checked={props.checked}
-      type={type}
-      data-test-id={testId}
-      onChange={(e) => {
-        if (onChange) {
-          onChange(e);
-        }
-      }}
-      onBlur={(e) => {
-        if (onBlur) {
-          onBlur(e);
-        }
-      }}
-      onFocus={(e) => {
-        if (onFocus) {
-          onFocus(e);
-        }
-      }}
-      aria-label={labelText}
-      id={id}
-      required={required}
-      disabled={disabled}
-      onKeyDown={handleKeyDown}
-      {...otherProps}
-    />
+    <div className={wrapperClassname}>
+      <input
+        className={classNames}
+        value={value}
+        name={name}
+        checked={props.checked}
+        type={type}
+        data-test-id={testId}
+        onChange={(e) => {
+          if (onChange) {
+            onChange(e);
+          }
+        }}
+        onBlur={(e) => {
+          if (onBlur) {
+            onBlur(e);
+          }
+        }}
+        onFocus={(e) => {
+          if (onFocus) {
+            onFocus(e);
+          }
+        }}
+        aria-label={labelText}
+        id={id}
+        required={required}
+        disabled={disabled}
+        onKeyDown={handleKeyDown}
+        {...otherProps}
+      />
+      {type === 'radio' ? (
+        /* eslint-disable-next-line jsx-a11y/label-has-associated-control */
```

Comment:
instead of this comment, I could add config, something like this https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/302#issuecomment-425512505

Reply:
Did you test the eslint config? Did you also test nesting the input inside the label?

Reply:
I can try to wrap input in label, but it would require a different CSS solution.

Reply:
let's wat for other people to give their opinion... I'm ok with doing like this for now :)